modules/testcontainers/build.gradle

```diff
@@ -3,6 +3,7 @@ version '1.0-SNAPSHOT'
 
 dependencies {
     api project(":client")
+    api project(":model")
```

Comment:
this should be redundant due `model` is available as transitive dependency of `client`